ffigen/README.md

```diff
@@ -0,0 +1,14 @@
+This package configures the ffigen to generate bindings to realm-core library.
+The is executed manually as needed and this package is never published.
+
+Usage: 
+
+On Windows: dart run ffigen --config windows.yaml
```

Comment:
Why do we have these 3 configs. The only change to the generated `realm-dart/ffigen/../lib/src/native/realm_bindings.dart` is a comment telling us what config was used, ie:
```
-/// A Windows config for ffigen Usage: dart run ffigen --config windows.yaml
+/// A Linux config for ffigen Usage: dart run ffigen --config linux.yaml
```

Reply:
these are host platform configurations and not target platform configs. They currently produce the same output yes.